shape-calculator/src/components/ShapeDisplay.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import '../styles/shapes.css'
+
+const ShapeDisplay = (props) => {
+    const shapeStyle = {
+        width: `${props.width}px`,
+        height: props.shape === 'square' || props.shape === 'circle' ? `${props.width}px` : `${props.height}px`,
+        background: `${props.color}`, 
+        borderRadius: props.shape === 'oval' ? props.width ? `${(props.width/2)}px` : '100px' / props.height ? `${(props.height/2)}px` : '50px' : ''
+    }
+
+    let shapeDisplay = props.shape ? <div className={props.shape} style={shapeStyle}>
```

Comment:
Not sure if this conditional render would be better inline 19.  You can probably use {props.shape &&}